inst/shiny/DiagnosticsExplorer/global.R

```diff
@@ -187,12 +187,6 @@ if (exists("database")) {
 
 if (exists("temporalTimeRef")) {
   temporalCharacterizationCovariateChoices <- get("temporalTimeRef") %>%
-    dplyr::filter((.data$startDay == -365 & .data$endDay == -31) |
```

Comment:
I'm not sure if this hard coding was intentional or not but it hides user settings that are likely important

Reply:
It was intentional. This makes the output match prior versions

Reply:
I disagree with opening up the diagnostics explorer to more temporal choices. We deliberately chose the time windows here. If we take this out, there is no difference between temporal-diagnostics and non temporal diagnostics as we have framed it.

Reply:
but we let the users choose these settings. It would *really really* annoy me if I made principled choices for, say, vaccine adverse events at risk windows (which takes time and discussion) then generated the results (which takes longer because of these settings) then I was unable to view them

Reply:
Yup. So these are conflicting ideas. 

The solution i have in my mind should work. Will show you